float64 linearVel
float64 angularVel
float64 angle
float64 x
float64 xd
float64 y
float64 yd
float64 ex
float64 ey
float64 evx
float64 evy
float64 eT

